tests/unit/oidc/test_views.py

```diff
@@ -546,7 +544,7 @@ def find_service(iface, **kw):
     assert macaroon_service.create_macaroon.calls == [
         pretend.call(
             "fakedomain",
-            f"OpenID token: fakespecifier ({datetime.fromtimestamp(0).isoformat()})",
+            f"OpenID token: example.yml ({datetime.fromtimestamp(0).isoformat()})",
```

Comment:
Is this change necessary?

Reply:
Related https://github.com/pypi/warehouse/pull/13980#discussion_r1448020448